src/timeline/InputBar.cpp

```diff
@@ -524,6 +524,8 @@ InputBar::command(QString command, QString args)
                 message(args, MarkdownOverride::ON);
         } else if (command == "plain") {
                 message(args, MarkdownOverride::OFF);
+        } else if (command == "rainbow") {
+                message(args, MarkdownOverride::NOT_SPECIFIED, true);
```

Comment:
```suggestion
                message(args, MarkdownOverride::ON, true);
```
Markup should always be on if you're going to post rainbow messages.

Reply:
Should it? Imo you could send a plain text message, that should get rainbowified, but I guess your solution fixes a few edge cases currently :3

Reply:
Well, what I'm thinking here is that basically we should always send rainbow messages with rich text enabled, even if we aren't parsing Markdown.

Reply:
Yes, but then the above change would be wrong. You always need to send richt text, that is true, but you may want to disable stuff being interpreted as markdown.

Reply:
I guess so.

Reply:
I agree w/ Nico here.  You might want to rainbowify something as-is without the markdown being interpreted.

Reply:
So what should i do about this?

Reply:
For now always specify ON, we can fix it later to also format non markdown text.